Sources/_StringProcessing/Engine/MEQuantify.swift

```diff
@@ -44,109 +44,114 @@ extension Processor {
 
   /// Generic quantify instruction interpreter
   /// - Handles .eager and .posessive
-  /// - Handles arbitrary minTrips and extraTrips
+  /// - Handles arbitrary minTrips and maxExtraTrips
   mutating func runQuantify(_ payload: QuantifyPayload) -> Bool {
```

Comment:
We should probably have assertions here:
- must not be reluctant
- minTrips >= 2

Reply:
Actually, no to the "should have hit a specialized path" assertion. Non-optimized builds are important for parity testing between the specialized and generic paths.